### ISSUE TITLE
citation for publications using this code?

### ISSUE THREAD

**Issue by stevendbrown** [naibaf7/PyGreentea#6]
is [this](https://arxiv.org/abs/1509.03371) the right document to cite if using this code in published data?

**Comment by naibaf7** [comment]
@stevendbrown 
Yes, this and the more recent IEEE ISBI paper: http://ieeexplore.ieee.org/document/7493487/?reload=true
